### PR TITLE
Update entities.rst

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

File: en/orm/entities.rst [hunk 1]
```diff
@@ -448,7 +448,7 @@ data. CakePHP makes this simple::
     // Convert to JSON
     $json = json_encode($user);
 
-When converting an entity to an array/JSON the virtual & hidden field lists are
+When converting an entity to an array/JSON the virtual & hidden field lists aren't
```
Comment (markstory) [inline en/orm/entities.rst]: But hidden fields are not output in array/json data.

**Comment by saeideng** [comment]
if you say this docs book is true,you can close the PR
thnks

**Comment by markstory** [comment]
@saeideng Have you observed otherwise?

**Comment by saeideng** [comment]
no at this time
if i find,i will open a new PR

**Pull request closed without merging**